packages/react-aria-components/src/Disclosure.tsx

```diff
@@ -71,7 +71,9 @@ export {_DisclosureGroup as DisclosureGroup};
 
 export interface DisclosureProps extends Omit<AriaDisclosureProps, 'children'>, RenderProps<DisclosureRenderProps>, SlotProps {
   /** An id for the disclosure when used within a DisclosureGroup, matching the id used in `expandedKeys`. */
-  id?: Key
+  id?: Key,
+  /** The children of the component. A function may be provided to alter the children based on component state. */
+  children: ReactNode | ((values: DisclosureRenderProps & {defaultChildren: ReactNode | undefined}) => ReactNode)
```

Comment:
ok this was probably overkill haha. I thought you were asking if you should make the children optional because they were already required, not if you should make them required.

Reply:
haha, it was the only way to get the property to show up with a meaningful type in the docs